src/base.ts

```diff
@@ -85,58 +40,78 @@ const getAuthorizationHeader = (accessToken = null) => {
   return { Authorization: `Bearer ${accessToken}` }
 }
 
-export interface RequestOptions {
-  isAuthorizedRequest?: boolean
+export const getHost = (host: string = null) => {
+  if (!apiClient.configuration.host) {
+    throw new Error(
+      'ApiClient not configured, please run: ApiClient.configure({ apiGatewayUrl: "your.api.gateway" }'
+    )
+  }
+
+  return host || apiClient.configuration.host
```

Comment:
what's the use case for passing a host to a function that is then returned transparently?

```
getHost("foo.local") // => foo.local
```

Reply:
The idea was to extract getting the host based on options. It returns the host (either passed one or the configured one) but also guards that the API client is configured

Reply:
that code still makes little sense to me even with your comment 🙈 
if it's just an internal thing, I'd roll with whatever you think is right. if it's external-facing, I'd get another opinion

Reply:
Yeah, everything here is internal. I'd rather have that separated from `fetch` wrapper since it makes things easier to grasp.